crates/aptos-in-memory-cache/src/caches/fifo.rs

```diff
@@ -0,0 +1,426 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{Cache, OrderedCache, StreamableOrderedCache};
+use dashmap::DashMap;
```

Comment:
add a comment on why dashmap?

Reply:
+1 to why dashmap? I thought there was a whole lotta other optimization to avoid it